app/assets/stylesheets/admins.css.scss

```diff
@@ -0,0 +1,137 @@
+
+
+// navigation
+#logo {
```

Comment:
create separate css files "header.scss|", "base.scss", ...

don't put all in a single SCSS 

CC @mmrshk